operator/docs/forwarding_logs_without_gateway.md

```diff
@@ -0,0 +1,124 @@
+# Forwarding Logs to LokiStack without LokiStack Gateway
+
+This document will describe how to send application, infrastructure, and audit logs to the Loki Distributor as different tenants using Fluentd or Vector.
+
+__Please read the [hacking guide](./hack_loki_operator.md) before proceeding with the following instructions.__
+
+_Note:_ This document is meant to be used with OpenShift only.
+
+_Disclaimer:_ This document helps in connecting fluentd or vector to LokiStack without authentication/authorization.
+
+## Openshift Logging
+
+[Openshift Logging](https://github.com/openshift/cluster-logging-operator) supports [forwarding logs to an external Loki instance](https://docs.openshift.com/container-platform/4.9/logging/cluster-logging-external.html#cluster-logging-collector-log-forward-loki_cluster-logging-external) using fluentd and vector as log forwarders.
+The below step-by-step guide will help you to send application, infrastructure, and audit logs to the LokiStack through the Distributor endpoint.
+The steps remain same for both fluentd and vector.
+
+In order to enable communication between the client(s) and the gateway, follow these steps:
+
+* Deploy the Loki Operator and a `lokistack` instance for [OpenShift](./hack_loki_operator.md#hacking-on-loki-operator-on-openshift).
+
+* Deploy the OpenShift Logging Operator from the Operator Hub or using the following command locally:
+
+    ```console
+    make deploy-image deploy-catalog install
+    ```
+  
+* Create a Cluster Logging instance in the `openshift-logging` namespace with only `collection` defined.
+
+    For fluentd:
+
+    ```yaml
+    apiVersion: "logging.openshift.io/v1"
+    kind: "ClusterLogging"
+    metadata:
+      name: "instance"
+      namespace: openshift-logging
+    spec:
+      collection:
+        logs:
+          type: "fluentd"
+          fluentd: {}
+    ```
+
+    For vector:
+
+    ```yaml
+    apiVersion: "logging.openshift.io/v1"
+    kind: "ClusterLogging"
+    metadata:
+      name: "instance"
+      namespace: openshift-logging
+    spec:
+      collection:
+        logs:
+          type: "vector"
+          fluentd: {}
+    ```
+
+* By default, the TLS communication is on for communication with the distributor. Therefore, you need to create a secret. The secret must exist in the openshift-logging namespace, and must have a key `ca-bundle.crt`.
```

Comment:
```suggestion
* By default, TLS is enabled on all components deployed by loki-operator. Because the service certificates are signed by a cluster-internal CA you need to set up a secret that enables the collector to validate the certificate returned by the distributor. The secret must exist in the openshift-logging namespace, and must have a key `ca-bundle.crt`.
```